src/internal/connector/graph/metadata_collection.go

```diff
@@ -88,7 +163,7 @@ func (md MetadataCollection) Items() <-chan data.Stream {
 	return res
 }
 
-// MetadataItem is an in-memory data.Stream implementation. MetadataItem does
+// metadataItem is an in-memory data.Stream implementation. MetadataItem does
```

Comment:
Not sure if this was meant to be changed to unexported or not
```suggestion
// MetadataItem is an in-memory data.Stream implementation. MetadataItem does
```

Reply:
Oh yeah, i went back and forth on that a few times.